### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_Zoom:Content disappears when zoom levels is 150% and above.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5964]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.4; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Additional Details:**
Tool Name: Zoom
MAS Reference:MAS1.4.4

**Steps to Reproduce:**
1. Install the app Azure Data Studio.
2. Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files.
3. Check the content on the page when Zoom is 100%.
4. Zoom page to 200% to check the content on the page.

**Actual:**
There is no scrolling functionality is available for the particular page.
**Similar issue observed in following pages:**
**1.MapDataSrcObjects:** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.

**Expected:**
Content should be visible when Page is Zoom up to 200%. 

**User Impact:​**
People with low vision will have difficulties in navigating to the menu Item bar when Zoom feature is used.

**Recommendations:​**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:​
MAS 1.4.4:** 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}](url)

Attachment for Reference:
[5964_[Accessibility]A11y_AzureDataStudio_CSVSelectDestinationDatbse_Zoom.pptx](https://github.com/microsoft/azuredatastudio/files/3274993/5964_.Accessibility.A11y_AzureDataStudio_CSVSelectDestinationDatbse_Zoom.pptx)

Does this issue occur when all extensions are disabled?: Yes
